app/_locales/en/messages.json

```diff
@@ -2875,7 +2875,7 @@
     "message": "Transaction encountered an error."
   },
   "transactionFee": {
-    "message": "Transaction Fee"
+    "message": "Transaction fee"
```

Comment:
`yarn test:unit:jest -u` is going to needed to be ran, and the changes committed, to update one of the swaps snapshots tests with this change.

Reply:
Also, the other tests this changes that need to be addressed are 
https://github.com/MetaMask/metamask-extension/blob/8ca0d24f8251ecc6ab915ebaae6929b060d07b5f/ui/pages/swaps/swaps-gas-customization-modal/swaps-gas-customization-modal.container.test.js#L22
and
https://github.com/MetaMask/metamask-extension/blob/95e3be4a85fdfc328e9365811f5972d99c7c0e48/ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.test.js#L61